src/Install/Installer.php

```diff
@@ -78,18 +79,29 @@ class Installer implements InstallerInterface
 	 */
 	private $databaseTableInstaller;
 
+	/**
+	 * @var Segment
+	 */
+	private $segment;
+
 	public function __construct(
 		Mollie $module,
 		ImageService $imageService,
-		InstallerInterface $databaseTableInstaller
+		InstallerInterface $databaseTableInstaller,
+		Segment $segment
 	) {
 		$this->module = $module;
 		$this->imageService = $imageService;
 		$this->databaseTableInstaller = $databaseTableInstaller;
+		$this->segment = $segment;
 	}
 
 	public function install()
 	{
+		$this->segment->setMessage('Mollie installed');
+		$this->segment->setOptions(['version' => $this->module->version]);
+		$this->segment->track();
+
```

Comment:
can we have version included always ? I mean inside segment factory since this information is unlikely to change